### DISCUSSION TITLE
Loose sensor on a lighthouse deck

### DISCUSSION THREAD

**Discussion by kleikemper** [bitcraze/discussions#1489] [Q&A] [answered]
After a crash the Lighthouse Deck did not receive any signals anymore and I realised that one of the sensors is not fully attached anymore.
I would like to get some guidance how to fix this. Thanks!
![cf_loose_sensor](https://github.com/user-attachments/assets/59626ccb-d4d9-4440-939c-0299cce21a69)

**Comment by knmcguire** [comment, marked as answer]
ah yes that is broken, but you should be able to fix it with some soldering, just make sure that the two pins on the left side are connected to the pad (check the right one for some reference, but you have to be careful not to damage the image sensor.

Also our experience that the signals won't be as good as before, but if it can initialize it again should be able to fly.
